spec/schedulers/TestScheduler-spec.ts

```diff
@@ -1,15 +1,14 @@
+/** @prettier */
```

Comment:
I hope this is the last PR where we have this line included. Can we update `.prettierrc.json` and completely remove `"overrides"` filed from it? 🤔

Reply:
@jakovljevic-mladen I created a draft PR that removes the pragma and the `spec` directory from the prettier formatting override. Please have a look at #7275. If we would like to merge those changes I can update the PR and promote it to a proper PR.

Reply:
> If we would like to merge those changes I can update the PR and promote it to a proper PR.

I think we do. Great @tmair, thanks a bunch!